src/AppBundle/Resources/views/Account/account.html.twig

```diff
@@ -113,6 +113,21 @@ activeTab: "tab-accounts"
 
         {{ form_end(form) }}
 
+    <script type="text/javascript">
+        var transactionHasChanged = false;
+        var message = "You haven't saved the transactions you entered. Do you want to leave without saving?";
```

Comment:
that could stay in the translation file (backslashing double quotes)

var message = '{{ 'file.key.subkey ' | trans | escape('js') }}'